.gitignore

```diff
@@ -1,2 +1,4 @@
 __pycache__
 models/
+samples
+core
```

Comment:
what is this?

Reply:
Tensorflow core dump file... probably not strictly necessary here either. I'll remove.